### DISCUSSION TITLE
uct_ib_mr_type_t

### DISCUSSION THREAD

**Discussion by EmmaLee-c** [openucx/ucx#8722] [Q&A] [answered]
typedef enum {
    /* Default memory region with either strict or relaxed order */
    UCT_IB_MR_DEFAULT,
    /* Additional memory region with strict order,
     * if the default region is relaxed order */
    UCT_IB_MR_STRICT_ORDER,
    UCT_IB_MR_LAST
} uct_ib_mr_type_t;

What does ‘UCT_IB_MR_STRICT_ORDER’ mean，why call ibv_reg_mr registration twice when UCX_IB_PCI_RELAXED_ORDERING=on?
Thanks!

**Comment by yosefe** [comment, marked as answer]
When need to send remote atomic operations, it should be done to a strict-pci-order memory key, to maintain atomic semantics
